src/main/resources/static/views/main/Navbar.html

```diff
@@ -25,13 +25,65 @@
                         </div>
                     </form>
                 </li>
-                <li><a href="#!/user">User (TEMPORARY)</a></li>
-            </ul>
-            <ul class="nav navbar-nav navbar-right">
-                <li class="active">
-                    <a ng-click="openPopup()">Login</a>
-                </li>
             </ul>
+            <div ng-show="!authenticated">
```

Comment:
You better use ng-if for such kind of things. It will remove content for unauthorised user from the DOM tree